tests/test_metrics.py

```diff
@@ -200,7 +203,11 @@ def test_cuda_available_model_no_device(self, model, is_available_mock):
         is_available_mock.return_value = True
         model.device = None
 
-        with pytest.warns(UserWarning):
+        with pytest.warns(
+            UserWarning,
+            match='``model.device`` attribute is ``None``. Since GPU is available, putting model '
+                  'on GPU.'
```

Comment:
Small formatting preference: 
```suggestion
            match=(
                '``model.device`` attribute is ``None``. Since GPU is available, putting model '
                'on GPU.'
            )
```

Reply:
Do you want me to change this for all of the tests in `test_model.py` too?

Reply:
oof... I guess you're right. Probably safest to just do it to everything that uses the `match` argument. 

Sorry about all this extra work 😬

Reply:
JK, all the `test_model.py` `match` arguments are one line except the `re.escape` ones.

Reply:
Even for those on one-line, I still think it's okay to just add the `re.escape` there just in case. If you want!